src/components/@shared/FormInput/TagsAutoComplete.tsx

```diff
@@ -74,10 +74,12 @@ export default function TagsAutoComplete({
       defaultValue={defaultTags}
       hideSelectedOptions
       isMulti
-      noOptionsMessage={() => 'Start typing to get suggestions'}
+      noOptionsMessage={() =>
+        'Start typing to get suggestions based on tags from all published assets.'
+      }
       onChange={(value: AutoCompleteOption[]) => handleChange(value)}
       onInputChange={(value) => handleOptionsFilter(tagsList, value)}
-      openMenuOnClick={false}
```

Comment:
@kremalicious just to give you some context: this was set to false because it was suggested in the comments not to show the complete list before the user has started to type something, to always show only relevant suggestions.

Reply:
interesting, when I tried it was just the difference on first click inside between this:

<img width="933" alt="Screen Shot 2022-09-27 at 14 18 49" src="https://user-images.githubusercontent.com/90316/192537295-6e5ac1f7-9150-4dea-9119-1f43863fdf2b.png">

and this

<img width="943" alt="Screen Shot 2022-09-27 at 14 17 30" src="https://user-images.githubusercontent.com/90316/192537089-e267f547-7169-4120-8f1b-de8d3b377fea.png">

As soon as I type 1 letter I get popup suggestion right away in both cases:

<img width="920" alt="Screen Shot 2022-09-27 at 14 19 29" src="https://user-images.githubusercontent.com/90316/192537428-7a148ec0-7dea-4e59-8816-52bb84b47b49.png">

So to make it more clear how the tags field can be used right away, opted to show and modify that help text to see expectations right away. Otherwise it is immediately confusing why I should `Create "d"`. That's for the UI, basically a suggestion of what the UI does and introduce all elements already to reduce surprises.

But agree that it stays a bit confusing as it looks like the tool in background needs at least 3 characters to produce anything meaningful:

<img width="881" alt="Screen Shot 2022-09-27 at 14 24 59" src="https://user-images.githubusercontent.com/90316/192538706-b618f6bd-0b67-49d6-ad21-0eb9db5221dc.png">

<img width="907" alt="Screen Shot 2022-09-27 at 14 25 09" src="https://user-images.githubusercontent.com/90316/192538774-8249b7e9-8c85-4cbd-a9aa-f3a0a338fb9c.png">

Both cases should give me suggestions already, as "data" contains "da". So this sounds more like we should prevent tag suggestions up until the 3rd character is typed, or modify how we use that `match-sorter`

Reply:
Yes, you are correct, it was to avoid opening an empty list when clicking on the input. Personally, I would prefer to show matches right after the first character has been typed (especially now that `match-sorter` has been implemented), but of course, that's up to you to decide 😄

Reply:
we can just change it to 2 chars, instead of 3, or 1 . change this `options={!input || input?.length < 3 ? [] : matchedTagsList}` to whatever we want. We can launch it with whatever and then collect feedback.

Reply:
works like every autosuggestion out there now with suggesting upon first key stroke. That `Create "xxx"` stays a bit confusing as it implies they are "created" upon click but in reality they are only "created" once publish finishes. But good enough